developers/weaviate/config-refs/env-vars/dynamic-config.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Runtime configuration management
+sidebar_label: Runtime configuration
+sidebar_position: 1
+image: og/docs/configuration.jpg
+---
+
+Weaviate supports dynamic configuration management, allowing you to update certain environment variables on the fly without restarting your instance. This feature helps you adapt settings in real time and fine-tune your instance based on evolving needs.
```

Comment:
> allowing you to update certain environment variables on the fly

Wdyt about: `allowing certain environment variables to be updated and read by Weaviate`